src/contact/containers/EditInfo.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import EditInfo from '../components/EditInfo';
+import * as actions from '../actions/Actions';
```

Comment:
Why import all actions?

Reply:
As I have worked on one module and I need my all actions here. I am using all.